docs/use_cases.md

```diff
@@ -244,3 +244,27 @@ can be used to constrain an insertion target for instance.
 
 If an anchor constrains a dispersion, the dispersion will be intrachromosomal even if the SV is set as interchromosomal 
 (In this case, other unconstrained dispersions of the SV will then be interchromosomal). 
+
+
+### Example 5b - Inversion within segmental duplications (SDs)
+To simulate INVs that span the regions between SDs, from a BED file annotating the SD regions.
+
+First, use the `workspace/complement_bed.py` to obtain the regions within SDs.
+Then, you can simulate `INV` script to generate a BED file containing the regions outside the SDs. 
+These are the "regions within SDs" you'll use for the inversion.
```

Comment:
how are the first/last segments handled? e.g. since we need to place between SDs, we should skip the first segment if it’s the start of the genome